master/master.cfg

```diff
@@ -154,7 +160,13 @@ build_factory.addStep(Git(repourl=zfs_repo, workdir="build/zfs",
     logEnviron=False, getDescription=True,
     lazylogfiles=True,
     description=["cloning"], descriptionDone=["cloned"]))
-
+build_factory.addStep(Git(repourl=lustre_repo, workdir="build/lustre",
+    mode="full", method="clobber", codebase="lustre",
```

Comment:
The full Lustre repository is large, we should do a shallow clone since we don't care about any of the history.  Add `shallow=True`.